packages/@ourworldindata/core-table/src/CoreTableColumns.ts

```diff
@@ -588,6 +588,18 @@ class BooleanColumn extends AbstractCoreColumn<boolean> {
     }
 }
 
+class OrdinalColumn extends CategoricalColumn {
+    @imemo get allowedValuesSorted(): string[] | undefined {
+        return this.def.sort
```

Comment:
I guess that `sort` does, by definition, always contain every single possible value, right?

Because I'm unsure what would happen if $\texttt{sort} \subsetneq \texttt{uniqNonEmptyStringVals}$.

Reply:
If that's the case, then we can safely ignore the second part of that question. Thanks.